DPG/Tasks/AOTTrack/qaEventTrack.cxx

```diff
@@ -184,6 +186,10 @@ struct qaEventTrack {
     histos.add("Tracks/Kine/phivspt", "#varphi vs #it{p}_{T}", kTH2F, {axisPt, axisPhi});
     if (doprocessMC || doprocessRun2ConvertedMC) {
       histos.add<TH2>("Tracks/Kine/resoPt", "", kTH2D, {axisDeltaPt, axisPt});
+      auto hreso = histos.add<TH2>("Tracks/Kine/resoInvPt", "", kTH2D, {axisDeltaPt, axisInvPt});
```

Comment:
here again: `axisDeltaPt` defines the residuals between the recon and gen pt, not 1/pt, right? If yes, one should change this axis as well

Reply:
I will use the same binning but different title